theseus_gui/src/components/RowDisplay.vue

```diff
@@ -0,0 +1,113 @@
+<script setup>
+import { ref, computed } from 'vue'
+import { ChevronLeftIcon, ChevronRightIcon } from 'omorphia'
+import Instance from './ui/Instance.vue'
+import News from './ui/News.vue'
+
+const props = defineProps({
+  instances: Array,
+  news: Array,
+  label: String,
+})
+
+const shouldRenderInstances = props.instances && props.instances?.length !== 0
+const shouldRenderNews = props.news && props.news?.length !== 0
+
+const handleLeftPage = () => {
+  console.log('page left')
+}
+
+const handleRightPage = () => {
+  console.log('page right')
+}
+</script>
+
+<template>
+  <div class="row">
+    <div class="header">
+      <p>{{ props.label }}</p>
+      <hr aria-hidden="true" />
+      <div class="pagination">
+        <ChevronLeftIcon @click="handleLeftPage" />
+        <ChevronRightIcon @click="handleRightPage" />
+      </div>
+    </div>
+    <section class="mods" v-if="shouldRenderInstances">
+      <Instance
+        v-for="instance in props.instances"
+        :key="instance.id"
+        display="gallery"
+        :instance="instance"
+      />
+    </section>
+    <section class="news" v-else-if="shouldRenderNews">
+      <News v-for="news in props.news" :key="news.id" :news="news" />
+    </section>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.row {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  margin-top: 2rem;
+  padding: 10px 0;
+
+  &:nth-child(odd) {
+    background: rgba(59, 59, 59, 0.5);
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: inherit;
+    width: 95%;
+
+    p {
+      font-size: 16px;
```

Comment:
use `1rem` instead, it's not in the current version of omorphia but adrian is adding the font variables soon